test/tst_atts.py

```diff
@@ -81,10 +82,17 @@ def setUp(self):
         v1.floatatt = FLOATATT
         v1.seqatt = SEQATT
         v1.stringseqatt = STRINGSEQATT
-        # issue #485 (triggers segfault in C lib 
+        # issue #485 (triggers segfault in C lib
         # with version 1.2.1 without pull request #486)
         f.foo = NP.array('bar','S')
         f.foo = NP.array('bar','U')
+        # issue #529 write string attribute as NC_CHAR unless
+        # it can't be decoded to ascii.  Add setncattr_string
+        # method to force NC_STRING.
+        f.charatt = u'foo' # will be written as NC_CHAR
+        f.setncattr_string('stringatt','bar') # NC_STRING
+        f.cafe = u'caf\xe9' # NC_STRING
+        v.setncattr_string('stringatt','bar') # NC_STRING
```

Comment:
Can we add an example writing bytes as an attribute? Seems like it's worth testing for completeness.